make/autoconf/toolchain.m4

```diff
@@ -43,6 +43,7 @@ VALID_TOOLCHAINS_macosx="gcc clang"
 VALID_TOOLCHAINS_bsd="gcc clang"
 VALID_TOOLCHAINS_aix="xlc"
 VALID_TOOLCHAINS_windows="microsoft"
+VALID_TOOLCHAINS_bsd="gcc clang"
```

Comment:
Call me blind, but this is already present in line 43, isn't it?

Reply:
Right, this looks like it is already there.

Reply:
good point as it is not present on the official repo nor in master I assumed wrongly it was here neither.